jax/_src/pallas/pallas_call.py

```diff
@@ -36,7 +36,8 @@
 from jax._src.interpreters import mlir
 from jax._src.interpreters import partial_eval as pe
 from jax._src.pallas import core as pallas_core
-from jax._src.pallas.primitives import uninitialized_value
+from jax._src.pallas import primitives
```

Comment:
lol, why is pyright erroring on this? Seems like something we should disable :)

Reply:
I changed that manually, because I need other APIs from `primitives.